### DISCUSSION TITLE
Is there a way to show chunk options in Quarto output?

### DISCUSSION THREAD

**Discussion by p0bs** [quarto-dev/quarto-cli#2311] [Q&A] [answered]
First things first: many thanks for Quarto, which is a real step forward.

Following on from [this issue in Rmarkdown](https://github.com/yihui/knitr/issues/1688#issuecomment-908565459), is there a chunk option, using the hash pipe, to show the hash pipes in Quarto output? (I realise that this is a bit meta!)

For example, let's say that I have the following chunks ...

```r
#| eval: false
#| echo: true

x <- long_process(a, b)
readr::write_rds(x, "x.rds")
```

... and ...

```r
x <- readr::read_rds("x.rds")

# remaining calculations here
```

... and I want to show that I've created `x` in the first chunk without evaluating it (as it is a long process to run). Moreover, I want to show that I've done this in the output, by showing the chunk options as part of the output. Is that possible via a specific setting? Or does it require a kluge?

Many thanks again ...

**Comment by cderv** [comment, marked as answer]
With Quarto, this would be the `echo: fenced` option. See in here: https://quarto.org/docs/computations/execution-options.html#fenced-echo

---

For example, here we demonstrate the use of the output and code-overflow options:

```{python}
#| echo: fenced
#| output: false
#| code-overflow: wrap
1 + 1
```

This code block appears in the rendered document as:

```{python}
#| output: false
#| code-overflow: wrap
1 + 1
```

---

Does it do what you are looking for ?

**Reply by p0bs** [reply]
Yes, that's perfect. Many thanks.
